tox.ini

```diff
@@ -9,12 +9,14 @@ deps=
   pytestrelease: pytest
   pytestmaster: git+https://github.com/pytest-dev/pytest.git@master
   pytestfeatures: git+https://github.com/pytest-dev/pytest.git@features
+  cython
 
 [testenv:benchmark]
-commands=py.test {posargs:testing/benchmark.py}
+commands=py.test testing/benchmark.py {posargs}
```

Comment:
it looks like this invocation got accidentally plucked incorrectly

Reply:
I actually did this on purpose so I could pass additional args without having to pass `testing/benchmark` every time. Should I put it back?